mpstat.c

```diff
@@ -2211,6 +2221,58 @@ void rw_mpstat_loop(int dis_hdr, int rows)
 	}
 }
 
+/*
+ ***************************************************************************
+ * Detect physical vCPU hotplug and call rw_mpstat_loop to display stats
+ *
+ * IN:
+ * @dis_hdr	Set to TRUE if the header line must always be printed.
+ * @rows	Number of rows of screen.
+ ***************************************************************************
+ */
+void mpstat_loop(int dis_hdr, int rows)
+{
+	while (1) {
+		photplug = FALSE;
+		/* Main loop for reading-writing stats */
+		rw_mpstat_loop(dis_hdr, rows);
```

Comment:
Test value returned by `rw_mpstat_loop()` function (see comment above).
Remove variable `photplug`.

Reply:
return value of rw_mpstat_loop() is tested and photplug variable removed